cmd/dranet/app.go

```diff
@@ -58,7 +58,7 @@ func init() {
 	flag.StringVar(&kubeconfig, "kubeconfig", "", "absolute path to the kubeconfig file")
 	flag.StringVar(&bindAddress, "bind-address", ":9177", "The IP address and port for the metrics and healthz server to serve on")
 	flag.StringVar(&hostnameOverride, "hostname-override", "", "If non-empty, will be used as the name of the Node that kube-network-policies is running on. If unset, the node name is assumed to be the same as the node's hostname.")
-	flag.StringVar(&celExpression, "filter", `attributes["dra.net/type"].StringValue  != "veth"`, "CEL expression to filter network interface attributes (v1.DeviceAttribute).")
+	flag.StringVar(&celExpression, "filter", `!("dra.net/type" in attributes) || attributes["dra.net/type"].StringValue  != "veth"`, "CEL expression to filter network interface attributes (v1.DeviceAttribute).")
```

Comment:
does this imply any visible change or is just a "nil check" to avoid the cel vm to error?

Reply:
👍

Reply:
@gauravkghildiyal but this is inconsistent with the goal, no? we want to ALWAYS publish the "real" devices no matter if they are used or not, if you ignore them and dranet restarts, those devices will never be published IIUIC